manifests/config/kubeadm.pp

```diff
@@ -366,8 +366,8 @@
         fail('Cannot use "cloud" as volume name')
       }
 
-      $apiserver_merged_extra_volumes = merge($apiserver_extra_volumes, $cloud_volume)
-      $controllermanager_merged_extra_volumes = merge($controllermanager_extra_volumes, $cloud_volume)
+      $apiserver_merged_extra_volumes = stdlib::merge($apiserver_extra_volumes, $cloud_volume)
```

Comment:
I think the function shouldn't be used. Instead use `+`.  That's Puppt native and doesn't require stdlib at all.

Reply:
@bastelfreak Thanks, that's definitely better. I'm not aware of any replacement for `to_yaml`

Reply:
for to_yaml we still need stdlib, yes.